createlinks

```diff
@@ -146,3 +146,13 @@ event_templates('pre-backup-config', qw(
     /etc/backup-config.d/nethserver-sssd.include
 ));
 
+
+#--------------------------------------------------
+# actions for account-provider-remove event
+#--------------------------------------------------
+
+event_actions('account-provider-remove', qw(
```

Comment:
The name of the UI page and docs use "Account*s* provider". I'd name the event after it: ``accounts-provider-remove``.